server/runtime.go

```diff
@@ -73,9 +73,19 @@ func (s *Server) CreatePodSandbox(ctx context.Context, req *pb.CreatePodSandboxR
 		logDir = fmt.Sprintf("/var/log/ocid/pods/%s", name)
 	}
 
-	// TODO: construct /etc/resolv.conf based on dnsOpts.
-	dnsOpts := req.GetConfig().GetDnsOptions()
-	fmt.Println(dnsOpts)
+	dnsServers := req.GetConfig().GetDnsOptions().GetServers()
```

Comment:
@mrunalp , @runcom , I did a test with a config file only including `name` field, and it still works.

    {
        "name": "podsandbox1"
    }